mindmeld/components/question_answerer.py

```diff
@@ -94,14 +165,55 @@ def _es_client(self):
             self.__es_client = create_es_client(self._es_host)
         return self.__es_client
 
-    @property
-    def _query_type(self):
-        if self._qa_config.get("model_type") in ALL_QUERY_TYPES:
-            return self._qa_config.get("model_type")
-        else:
-            return DEFAULT_QUERY_TYPE
+    def _load_field_info(self, index):
+        """load knowledge base field metadata information for the specified index.
+
+        Args:
+            index (str): index name.
+        """
+
+        # load field info from local cache
+        index_info = self._es_field_info.get(index, {})
+
+        if not index_info:
+            try:
+                # TODO: move the ES API call logic to ES helper
```

Comment:
It would be good to finish this todo in this PR

Reply:
It's a TODO from previous version of code. Sure, I'll try to see how to resolve it.

Reply:
@mhuang2173, @vijay120, @vembar :  I am not fully sure what part of code is required to me moved to helpers from here. Can you take a quick look and give a suggestion. Thanks!

Reply:
Ah got it, if so, thats OK (ie doesnt need to be completed this PR)